opal/scaffolding/scaffold/app/settings.py.jinja2

```diff
@@ -150,108 +78,154 @@ TEMPLATES = [
                 'opal.context_processors.settings',
                 'opal.context_processors.models',
                 'opal.core.pathway.context_processors.pathways',
-
             ],
-
-
-            # ... some options here ...
+            'loaders': [(
+                # TODO: Django 1.11 makes turns this loader on by default, so
+                # this section and `debug` can be removed then.
+                'django.template.loaders.cached.Loader', [
+                    'django.template.loaders.filesystem.Loader',
+                    'django.template.loaders.app_directories.Loader',
+                ],
+            )],
         },
     },
 ]
 
+WSGI_APPLICATION = '{{ name }}.wsgi.application'
 
-INSTALLED_APPS = (
-    'django.contrib.auth',
-    'django.contrib.contenttypes',
-    'django.contrib.sessions',
-    'django.contrib.sites',
-    'django.contrib.messages',
-    'django.contrib.staticfiles',
-    'axes',
-    'reversion',
-    'rest_framework',
-    'rest_framework.authtoken',
-    'compressor',
-    'opal',
-    'opal.core.search',
-    'opal.core.pathway',
-    'opal.core.referencedata',
-    '{{ name }}',
-    'django.contrib.admin',
-)
+# Database
+# https://docs.djangoproject.com/en/{{ docs_version }}/ref/settings/#databases
+try:
+    import dj_database_url
 
-# A sample logging configuration. The only tangible logging
-# performed by this configuration is to send an email to
-# the site admins on every HTTP 500 error when DEBUG=False.
-# See http://docs.djangoproject.com/en/dev/topics/logging for
-# more details on how to customize your logging configuration.
-LOGGING = {
-    'version': 1,
-    'disable_existing_loggers': False,
-    'filters': {
-        'require_debug_false': {
-            '()': 'django.utils.log.RequireDebugFalse'
+    DATABASES = {
+        'default': dj_database_url.config(default='sqlite:///' + PROJECT_PATH + '/opal.sqlite')
+    }
+except ImportError:
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.sqlite3',
+            'NAME': os.path.join(PROJECT_PATH, 'opal.sqlite'),
+            'USER': '',
+            'PASSWORD': '',
+            'HOST': '',
+            'PORT': ''
         }
+    }
+
+# Password validation
+# https://docs.djangoproject.com/en/{{ docs_version }}/ref/settings/#auth-password-validators
+AUTH_PASSWORD_VALIDATORS = [
+    {
+        'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator',
```

Comment:
can we double check that this doesn't break

https://github.com/openhealthcare/opal/blob/84b809a4a29b84e97730018de9e49517f2564ef8/opal/scaffolding/scaffold/app/management/commands/createopalsuperuser.py.jinja2

It shouldn't because the docs say they get validated in the form.

In dango 2 though some validation also get's applied on the command line so can we check...